src/MLUtils.jl

```diff
@@ -22,7 +22,11 @@ import NNlib
 @traitdef IsTable{X}
 @traitimpl IsTable{X} <- Tables.istable(X)
                       
-
+if VERSION < v"1.9.0-DEV.1163"
+    import Compat: stack
+else
+    import Base: stack
+end
```

Comment:
Can simplify, as this will get Base version when available:
```suggestion
using Compat: stack
```